features/step_definitions/route.rb

```diff
@@ -112,3 +112,22 @@
   cb[cb_name] = res[:response].scan(/^I\d+.*Router reloaded:$/).last
   logger.info "The last reloaded log with timestamp is: #{cb[cb_name]}."
 end
+
+Given /^I use the router project$/ do
+  if env.version_ge("4.0", user: user)
+    step %Q/I use the "openshift-ingress" project/
+  else
+    step %Q/I use the "default" project/
+  end
+end
+
+Given /^a router pod becomes ready with default labels$/ do
```

Comment:
I think better step would be `all default router pods become ready`. Otherwise it is possible clients hit a non-ready pod thus instability.

Reply:
Updated, thanks.